diff --git a/node_modules/@cesium/engine/Source/Core/buildModuleUrl.js b/node_modules/@cesium/engine/Source/Core/buildModuleUrl.js
index 8d99d2b..88a1982 100644
--- a/node_modules/@cesium/engine/Source/Core/buildModuleUrl.js
+++ b/node_modules/@cesium/engine/Source/Core/buildModuleUrl.js
@@ -41,7 +41,7 @@ function getCesiumBaseUrl() {
   let baseUrlString;
   if (typeof CESIUM_BASE_URL !== "undefined") {
     baseUrlString = CESIUM_BASE_URL;
-  } else if (defined(import.meta) && defined(import.meta.url)) {
+  } else if (defined(import.meta?.url)) {
     // ESM
     baseUrlString = getAbsoluteUri(".", import.meta.url);
   } else if (
